drizzle-orm/src/column-builder.ts

```diff
@@ -1,5 +1,6 @@
 import { entityKind } from '~/entity.ts';
 import type { Column } from './column.ts';
+import type { MsSqlColumn } from './mssql-core/index.ts';
```

Comment:
Looks like a duplicate import on this line.
Apologies not familiar with git and unable to lodge a change myself.